src/scss/components/_resources.scss

```diff
@@ -127,3 +127,9 @@
     content: ' | ';
   }
 }
+
+[data-list='tag'] {
+  &:last-of-type {
```

Comment:
While I think `data-list='tag'` would probably be enough, I added `last-of-type` in case we ever stack these. The goal was to give the footer a little more breathing room on pages with the lists, and not on the other pages, like the homepage which already has plenty of space.